test/functional/hooks.js

```diff
@@ -42,37 +43,126 @@ describe('library hooks', () => {
     }
 
     beforeEach(() => {
-      sandbox.spy(attributes, 'applyAttr');
+      stub = sinon.stub();
```

Comment:
So, the `staticName` and `dynamicName` still have to use stubs since you can't spy on a non-function. But I've updated to use sandbox where I can.